apps/ledger-live-desktop/src/renderer/actions/settings.ts

```diff
@@ -254,14 +247,16 @@ export const showToken = (tokenId: string) => ({
   type: "SHOW_TOKEN",
   payload: tokenId,
 });
-export const unhideNftCollection = (collectionId: string) => ({
-  type: "UNHIDE_NFT_COLLECTION",
-  payload: collectionId,
-});
-export const unwhitelistNftCollection = (collectionId: string) => ({
-  type: "UNWHITELIST_NFT_COLLECTION",
-  payload: collectionId,
+
+export const updateNftStatus = (
+  blockchain: BlockchainsType,
+  collectionId: string,
+  status: NftStatus,
+) => ({
+  type: "UPDATE_NFT_COLLECTION_STATUS",
```

Comment:
suggestion : you can extract it to `actions/constants.ts` as 
an event based action type UPDATE_NFT_COLLECTION_STATUS = `settings/updateNftCollectionStatus`